libraries/AP_Notify/Buzzer.cpp

```diff
@@ -29,7 +29,7 @@ bool Buzzer::init()
         return false;
     }
     _pin = pNotify->get_buzz_pin();
```

Comment:
I think you also need to change `get_buzz_pin` to return `int8_t`

Reply:
duh...yep..done